docs/configuration.rst

```diff
@@ -37,6 +37,12 @@ Core
                                          used if the password hash type is set
                                          to something other than plain text.
                                          Defaults to ``None``.
+``SECURITY_SINGLE_HASH``                 Specifies that passwords should only be
```

Comment:
Can you please include `PASSWORD_` in the config name since it's related to password hashing?